### ISSUE TITLE
the loss is so big

### ISSUE THREAD

**Issue by pzz2011** [twtygqyy/pytorch-SRDenseNet#4]
Epoch[1](100/274): Loss: 116810211328.0000000000=>is it an error?

I just use dataset Yang 91 as the training dataset.

**Comment by twtygqyy** [comment]
Hi @pzz2011 the loss looks normal, since the training is just started. You can wait for a couple of epochs and see the drop.

**Comment by pzz2011** [comment]
thanks.
